.github/workflows/post-merge-dynamo-to-dev.yml

```diff
@@ -21,7 +21,8 @@ jobs:
     runs-on: ubuntu-latest
     defaults:
       run:
-        working-directory: infra-l2-dynamo/**
+        working-directory: infra-l2-dynamo
+        template-file: template.yaml
```

Comment:
let's remove this `template-file` default.
wont work in the run command below, and best to set it for the action so it's easier to see.